assets/app/lib/settings.rb

```diff
@@ -21,6 +21,7 @@ module Settings
     SETTINGS = {
       notifications: true,
       simple_logos: false,
+      alternate_spreadsheet: false,
```

Comment:
No need for an option. The dotted underlining doesn’t really work with linked actions anyway, so please revise this PR, get rid of dotted underlining and implement your changes without user choice.

Reply:
I removed the choice to use the old style